README.md

```diff
@@ -187,13 +187,14 @@ capable stylesheet transformer.
 
 ### Supported Transformations
 
-| From ╲ To           | hOCR | ALTO | PAGEXML | FineReader | Google Cloud Vision |
-| ---:                | ---  | ---  | ---     | ---        | ---                 |
-| hOCR                | =    | ✓    | -       | -          | -                   |
-| ALTO                | ✓    | =    | ✓       | -          | -                   |
-| PAGEXML             | ✓    | ✓    | =       | -          | -                   |
-| FineReader          | ✓    | -    | -       | =          | -                   |
-| Google Cloud Vision | ✓    | -    | -       | -          | =                   |
+| From ╲ To           | hOCR | ALTO | PAGEXML | FineReader |
```

Comment:
Why not remove the FineReader column, too? Or keep the Google Cloud Vision column?

Reply:
I am not aware of any transformation to Google Cloud Vision format nor would I see any reason to work on that. The FineReader format might be more common, but actually the same argument might hold for them. Thus, I would be fine to delete the column as well. :+1: Maybe, it is more realistic to add some transformation to TEI at some point, but we can add such a column, when we need it.